src/pages/events/index.tsx

```diff
@@ -86,14 +86,12 @@ const Events: React.FC = () => {
   };
 
   return (
-    <div className="events-page">
-      <div className="events-header">
+    <div className="events">
```

Comment:
Events div has a white bar on the left 
![image](https://user-images.githubusercontent.com/8893880/95513291-f6681180-096e-11eb-9ca8-a8043b27a62a.png)